src/tests.rs

```diff
@@ -259,7 +402,6 @@ fn test_rpm_header_base(package: RPMPackage) -> Result<(), Box<dyn std::error::E
         assert_eq!(*tag, metadata.signature.index_entries[i].tag);
     }
 
-    assert_eq!("cpio", metadata.get_payload_format()?);
```

Comment:
Tbh, never encountered a single rpm file in the last decade where the content was not a CPIO archive :sweat_smile:

Reply:
Technically all of them produced in the past decade (with recent versions of RPM) are not CPIO archives, the payload format tag is just lying.

https://github.com/rpm-software-management/rpm/issues/2447

Reply:
Oh ffs..